addons/l10n_es/data/template/account.tax-es_common.csv

```diff
@@ -213,11 +213,11 @@
 "","","","","","","","","","","","","tax","l10n_es.mod_303_29","invoice","account_common_472","",""
 "","","","","","","","","","","","","base","l10n_es.mod_303_40","refund","","",""
 "","","","","","","","","","","","","tax","l10n_es.mod_303_41","refund","account_common_472","",""
-"account_tax_template_s_iva0","","VAT Exempt","VAT Exempt","sale","0% EXEMPT","IVA Exento Repercutido Sujeto","0.0","percent","tax_group_iva_0","","","base","","invoice","","","IVA Exento"
+"account_tax_template_s_iva0","","VAT Exempt","VAT Exempt","sale","0% EXEMPT","IVA Exento (Otros)","0.0","percent","tax_group_iva_0","","","base","","invoice","","","IVA Exento"
 "","","","","","","","","","","","","tax","","invoice","account_common_472","",""
 "","","","","","","","","","","","","base","","refund","","",""
 "","","","","","","","","","","","","tax","","refund","account_common_472","",""
-"account_tax_template_s_iva0_ns","","VAT Exempt not subjected","VAT Exempt Non-Subject","sale","0% EXEMPT NS","IVA Exento Repercutido No Sujeto","0.0","percent","tax_group_iva_0","","","base","","invoice","","","IVA Exento No Sujeto"
+"account_tax_template_s_iva0_ns","","VAT Exempt not subjected","VAT Exempt Non-Subject","sale","0% EXEMPT NS","IVA Exento Sin Derecho A Deducción","0.0","percent","tax_group_iva_0","","","base","","invoice","","","IVA Exento Sin Derecho A Deducción"

```

Comment:
This tax is ignored when sending, so it should be "IVA no aplicable" in Spanish and VAT Not Applicable in English.

Reply:
@rafaelbn I tend to agree with @jco-odoo remark here.

Reply:
I didn't change "0% EXEMPT NS" yet, is it also relevant to change?

Reply:
We agree [here](https://github.com/odoo/odoo/issues/124942#issuecomment-1618655448)

> Tax l10n_es.account_tax_template_s_iva0_ns | "IVA Exento Repercutido No Sujeto"
> ​=> Rename to IVA Exento sin derecho a deducción

Is it @chklop @jco-odoo ?

Reply:
> Tax l10n_es.account_tax_template_s_iva0_ns | "IVA Exento Repercutido No Sujeto"
​=> Rename to IVA Exento sin derecho a deducción

@jco-odoo because is Art. 20 and is "Casilla 83" in 303. I showed you this:

![imagen](https://github.com/odoo/odoo/assets/8736623/bbb7c1cf-bbfc-4a6c-a092-40addce3d6a5)

Ref: https://sede.agenciatributaria.gob.es/Sede/todas-gestiones/impuestos-tasas/iva/modelo-303-iva-autoliquidacion_/instrucciones.html

Reply:
@rafaelbn Thanks again!  We should add the tag for the casilla 83 as well then.

Reply:
> Thanks again! We should add the tag for the casilla 83 as well then.

@jco-odoo no, remember (Valencia is far away 😄 , because casilla 83 is only for the LAST period of the year and not quarterly or monthly. This is a really special use case.

Reply:
@rafaelbn I remember that.  It is just that I thought the report would need to filter it out.  (based on the period)